docs/FOLDERS.md

```diff
@@ -24,37 +24,47 @@
 Contains:
  1. Actions, scripts that interact with the services in [services](#services) folder.
  2. Endpoints, exposing the service to other services.
+ 3. Handles the validations and assertions of the requests
 
 ### api/common
```

Comment:
in the common u have have something like middlewares too

Reply:
the middlewares are express middlewares that can use used by any sub router

Reply:
added that